packages/sdk/src/protocols/lightpush/index.ts

```diff
@@ -89,16 +97,23 @@ class LightPushSDK extends BaseProtocolSDK implements ILightPushSDK {
           successes.push(success);
         }
         if (failure) {
+          failures.push(failure);
```

Comment:
leaving for some context, will make an issue for it later so that we can discuss 

in status-go where retry is already implemented they do things differently and we will probably do something similar too

when message is sent it is queued, then it will be retried and once it is definitely fails - it will be communicated to consumer through something similar to event API 

cc @waku-org/js-waku

Reply:
decoupled into follow up https://github.com/waku-org/js-waku/issues/2140